dev-tools/omdb/tests/test_all_output.rs

```diff
@@ -38,12 +38,14 @@ async fn test_omdb_usage_errors() {
         &["db"],
         &["db", "--help"],
         &["db", "dns"],
-        &["db", "dns", "show"],
```

Comment:
I removed this test because the point was to show that we produced a useful error when you haven't provided any URLs.  But that's not an error any more.